scripts/init.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+set -euo pipefail
+IFS=$'\n\t'
+
+packageName=${1:-}
+
+ROOT_PACKAGE_NAME=$(jq -r ".name" < package.json)
+if [ "$ROOT_PACKAGE_NAME" != "lisk-elements-monorepo" ]; then
+	echo "Please use the command in the project root directory."
+	exit 1;
+fi
+
+if [ -z "$packageName" ] || [[ "$packageName" =~ [^a-zA-Z0-9-] ]]; then
+	echo "Usage: npm run init -- PACKAGE_NAME"
+	exit 1
+fi
+
+packageDir="./packages/$packageName"
+# Just in case package folder doesn't exist yet.
+mkdir -p "$packageDir"
+
+files=(
```

Comment:
`sources` is maybe more accurate since this includes directories.

Reply:
I agree files is not a good choice but then I prefer to use `templates`.